doc/MANUAL.adoc

```diff
@@ -1142,6 +1142,7 @@ The default is *subdirs*.
 === Redis storage backend
 
 URL format: `+redis://[[USERNAME:]PASSWORD@]HOST[:PORT][/DBNUMBER]+`
+Alternative: `+redis+unix://SOCKET_PATH[?db=DBNUMBER[&password=PASSWORD]]+`
```

Comment:
I suggest letting db and password be attributes instead of URL parameters (e.g. `redis+unix:///example/path|db=0|password=example`). That way the attributes can apply to both redis and redis+unix (overriding URL parameters), and it's easier to redact the password since the framework already supports that. Does that sound OK to you?

Reply:
I think the syntax was borrowed from perl: https://metacpan.org/pod/URI::redis

But they _could_ be attributes ("secret") instead of part of the URL (in user_info)

That would probably apply to all forms of URL, though ?

i.e. also to the regular (non-unix) http and redis, right ?

Reply:
> I think the syntax was borrowed from perl: [metacpan.org/pod/URI::redis](https://metacpan.org/pod/URI::redis)

I see. I like the `redis+unix://[:PASSWORD@]SOCKET_PATH[?db=DATABASE]` variant. Perhaps support only that one?

Reply:
Ok, that should be more similar to the other backends

Reply:
When using user info, a host is required.

So syntax is: `redis+unix://[[USERNAME:]PASSWORD@localhost]SOCKET_PATH[?db=DBNUMBER[&password=PASSWORD]]`

Reply:
Removed the code for password parameter